### PR TITLE
This fixes #150

### LINKED ISSUE #150: Bring thank you text in the center
On /submit_contest/ page, bring the text in the center of the page i.e., both horizontally and vertically in the center of the page.
thank you page

Also replace *Y* with *y*.

### PR DESCRIPTION
**Pull request by rudr4x5h**
**Checklist**

- [x] My branch is up-to-date with the upstream `develop` branch.
- [ ] I have added necessary documentation (if appropriate).

**Which issue does this PR fix?**: fixes #150

### CODE REVIEW

**Review by vaibhavsingh97 — APPROVED** [review]
LGTM!! 🚀

File: oshc/main/templates/contest_submission.html [hunk 1]
```diff
@@ -4,10 +4,10 @@
 
 <div class="jumbotron text-xs-center">
   <div class="container text-center">
-      <h1 class="display-3">Thank You!</h1>
-      <p class="lead"><strong>Your contest has been submitted for review.</strong></p>
+      <h1 class="display-3 text-center">Thank you!</h1>
+      <p class="lead text-center"><strong>Your contest has been submitted for review.</strong></p>
```
Comment (tapaswenipathak) [inline oshc/main/templates/contest_submission.html]: This isn't working on my local or on a trial deployment. Can you please share screenshots?

**Review by tapaswenipathak — CHANGES_REQUESTED** [review]

File: oshc/main/templates/contest_submission.html (hunk 1 above)
Comment (rudr4x5h) [reply oshc/main/templates/contest_submission.html]: Sure ma'am, here's the screenshot of my local deployment.
local_oshc

Comment (tapaswenipathak) [reply oshc/main/templates/contest_submission.html]: If you check this is what the current status is. 

I added few more words in the description of the issue. By center I meant, vertically and horizontally in the center of the page, somewhere in the middle.

Comment (rudr4x5h) [reply oshc/main/templates/contest_submission.html]: @tapasweni-pathak it's done, have a look.
done_oshc

File: oshc/main/static/main/css/app.css [hunk 2]
```diff
@@ -402,3 +402,21 @@ footer .container {
...
+  z-index: -100;
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+}
+
+.subtitle {
+  margin: 0;
+}
```
Comment (tapaswenipathak) [inline oshc/main/static/main/css/app.css]: 👍 

Can you please squash the commits into one, then we are good to go with this one! 🚀

**Review by tapaswenipathak — CHANGES_REQUESTED** [review]

File: oshc/main/templates/contest_submission.html [hunk 3]
```diff
@@ -3,11 +3,11 @@
...
 <div class="jumbotron text-xs-center">
-  <div class="container text-center">
-      <h1 class="display-3">Thank You!</h1>
-      <p class="lead"><strong>Your contest has been submitted for review.</strong></p>
+  <div class="container text-center vcenter">
+      <h1 class="display-3 text-center">Thank you!</h1>
+      <p class="lead text-center subtitle"><strong>Your contest has been submitted for review.</strong></p>
       <hr>
-      <p class="lead">
+      <p class="lead text-center">
```
Comment (tapaswenipathak) [inline oshc/main/templates/contest_submission.html]: Also you can remove all ```text-center``` but on the first div here.

**Review by tapaswenipathak — CHANGES_REQUESTED** [review]

File: oshc/main/static/main/css/app.css (hunk 2 above)
Comment (rudr4x5h) [reply oshc/main/static/main/css/app.css]: @tapasweni-pathak done 👍

File: oshc/main/templates/contest_submission.html (hunk 3 above)
Comment (rudr4x5h) [reply oshc/main/templates/contest_submission.html]: @tapasweni-pathak done 👍

File: oshc/main/static/main/css/app.css (hunk 2 above)
Comment (tapaswenipathak) [reply oshc/main/static/main/css/app.css]: Not yet :( [Screenshot from Gyazo](https://gyazo.com/d7f6311aa8699e6dcb7d33db9a8e26df). Add a commit message *Move confirmation text to the center (v,h)*.

Comment (rudr4x5h) [reply oshc/main/static/main/css/app.css]: @tapasweni-pathak check it now 😄

File: oshc/main/templates/contest_submission.html [hunk 4]
```diff
@@ -3,9 +3,9 @@
 {% block content %}
 
 <div class="jumbotron text-xs-center">
-  <div class="container text-center">
-      <h1 class="display-3">Thank You!</h1>
-      <p class="lead"><strong>Your contest has been submitted for review.</strong></p>
+  <div class="container text-center vcenter">
+      <h1 class="display-3 text-center">Thank you!</h1>
```
Comment (tapaswenipathak) [inline oshc/main/templates/contest_submission.html]: These ```text-center```s came back.  :) Only the first one is required.

**Review by tapaswenipathak — CHANGES_REQUESTED** [review]

File: oshc/main/templates/contest_submission.html (hunk 4 above)
Comment (rudr4x5h) [reply oshc/main/templates/contest_submission.html]: @tapasweni-pathak fixed 👍

**Comment by rudr4x5h** [comment]
Finally 😄 👍

**Pull request merged**